### DISCUSSION TITLE
Восстановление работоспособности

### DISCUSSION THREAD

**Discussion by Kirdykun** [Anonym-tsk/nfqws-keenetic#41] [Делимся опытом]
12.09.24 перестало работать - есть превью, есть коменты, само виде не грузится.
Помогло в конфиге /opt/etc/nfqws/nfqws.conf вместо  --dpi-desync=fake,disorder2 оставить --dpi-desync=disorder2

**Comment by Anonym-tsk** [comment]
Обновитесь на версию 2.1.1

**Reply by walkman99** [reply]
Как корректно обновить? Установкой или есть какая-то команда?
